.github/workflows/build-and-publish-chart.yml

```diff
@@ -73,6 +79,14 @@ jobs:
           git commit -m "Update Chart.yaml to version ${{ env.NEW_VERSION }}"
           git push origin main
 
+      # This must happen after changes to the main branch
+      - name: Create the Github Tag Version
```

Comment:
it would be nice if we could tag it with the`steps.nexttag.outputs.patch` output, so that the Chart.yaml and tag version will always be the same. if I understand correctly, this gets the next tag version, updates the chart.yaml with it, and then uses this step to get update the tag. I guess they should always be the same if things are working correctly, but we're getting the same value from two different places, right?

Reply:
Actually - I was thinking in `main` they should be the same. In `develop` we may want only `develop`. SInce this is fed to the validator in the configMap, we can know if a customer deployed something half baked...